stable/influxdb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: influxdb
-version: 1.4.0
+version: 1.5.0
```

Comment:
Thanks for your PR, so cleanly. But there may have some considerations on the chart version. This PR include don't backward compatibility. We should bump the chart version to `2.0.0` and make tips on README.md.

Reply:
Totally agree with you. I'm going to work on that soon!